riotfile.py

```diff
@@ -291,9 +291,6 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                     command="pytest {cmdargs} tests/tracer/ --ignore=tests/tracer/test_http.py",
                 ),
             ],
-            env={
-                "DD_REMOTE_CONFIGURATION_ENABLED": "false",
-            },
```

Comment:
we need this env var because Remote Config creates conflicts with `tests/tracer/`